src/calculateRentalCost.js

```diff
@@ -29,6 +29,11 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rent = 40;
+
+  return days < 3 ? days * rent
+    : days < 7 ? days * rent - 20
+      : days * rent - 50;
```

Comment:
don't use ternary operator for conditions with more than one condition. In this case it is better to use if..else or switch..case